packages/hardware-trezor/src/transformers/keyPaths.ts

```diff
@@ -16,6 +16,21 @@ export const paymentKeyPathFromGroupedAddress = (address: GroupedAddress): BIP32
   address.index
 ];
 
+export const stakeKeyPathFromGroupedAddress = (address: GroupedAddress | undefined): BIP32Path | null => {
```

Comment:
Let's simplify the conditions in this function with the safe navigation operator:
```ts
  if (!address?.stakeKeyDerivationPath) return null;
  return [
    util.harden(CardanoKeyConst.PURPOSE),
    util.harden(CardanoKeyConst.COIN_TYPE),
    util.harden(address.accountIndex),
    address.stakeKeyDerivationPath.role,
    address.stakeKeyDerivationPath.index
  ];
```

Reply:
✅